.github/workflows/mac.yml

```diff
@@ -48,10 +45,6 @@ on:
         default: ""
         required: false
         type: string
-      wpt-layout:
```

Comment:
We shouldn't remove this parameter. They will be updated as part of a different issue.